src/core/installAddon.js

```diff
@@ -46,8 +46,23 @@ import {
 import * as addonManager from 'core/addonManager';
 import {
   USER_AGENT_OS_ANDROID,
+  USER_AGENT_OS_BSD_FREEBSD,
+  USER_AGENT_OS_BSD_NETBSD,
+  USER_AGENT_OS_BSD_OPENBSD,
+  USER_AGENT_OS_BSD_PC,
   USER_AGENT_OS_LINUX,
+  USER_AGENT_OS_LINUX_ARCH,
+  USER_AGENT_OS_LINUX_CENTOS,
+  USER_AGENT_OS_LINUX_DEBIAN,
+  USER_AGENT_OS_LINUX_FEDORA,
+  USER_AGENT_OS_LINUX_GENTOO,
+  USER_AGENT_OS_LINUX_GNU,
```

Comment:
[But aren't *all* Linuxes GNU? Stallman would be angry.](https://www.gnu.org/gnu/linux-and-gnu.en.html) 😉

Reply:
😈